reports/Ali-Reza/Exercise/Exercise.md

```diff
@@ -0,0 +1,21 @@
+# Exercise
+
+**These are the days that I do Exercise**
+- [ ] Monday

```

Comment:
@AliReza1083 I think the position of x in [x] is reversed. If you're done with Monday, put [x] inside, it will be checked

Reply:
It must be empty @avie-dev, Because I don't do exercise on Monday.

Reply:
ooh ok